samples/Uno.Toolkit.Samples/Uno.Toolkit.Samples.Shared/Converters/HexToColorConverter.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System;
+
+using Windows.UI;
+
+#if IS_WINUI
+using Microsoft.UI.Xaml.Data;
+#else
+using Windows.UI.Xaml.Data;
+#endif
+
+namespace Uno.Toolkit.Samples.Converters
+{
+	public class HexToColorConverter : IValueConverter
+	{
+		public object Convert(object value, Type targetType, object parameter, string language)
+		{
+			return value.ToString();
+		}
+
+		public object ConvertBack(object value, Type targetType, object parameter, string language)
+		{
+			if (value is not string { Length: 9 } hex || !hex.StartsWith("#"))
```

Comment:
[`XamlBindingHelper.ConvertValue`](https://github.com/unoplatform/uno/blob/3fcc7128029530ea9c27c8d69cfb8d5ea7185a65/src/Uno.UI/UI/Xaml/Markup/XamlBindingHelper.cs#L21C16-L21C16) is able to peform this conversion (see [this](https://github.com/unoplatform/uno/blob/3fcc7128029530ea9c27c8d69cfb8d5ea7185a65/src/Uno.UWP/UI/Colors.cs#L30)).